core/src/avm2/globals/flash/display/NativeWindow.as

```diff
@@ -0,0 +1,314 @@
+    }
+
+    // Triggers a visual cue through the operating system that an event of interest has occurred.
+    public function notifyUser(type:String):void
+    {
+      stub_method("flash.display.NativeWindow", "notifyUser");
+    }
+
+    // Sends this window directly behind the specified window.
+    public function orderInBackOf(window:NativeWindow):Boolean
+    {
+      stub_method("flash.display.NativeWindow", "orderInBackOf");
+      return false;
+    }
+
+    // Brings this window directly in front of the specified window.
+    public function orderInFrontOf(window:NativeWindow):Boolean
+    {
+      stub_method("flash.display.NativeWindow", "orderInFrontOf");
+      return false;
+    }
+
+    // Sends this window behind any other visible windows.
+    public function orderToBack():Boolean
+    {
+      stub_method("flash.display.NativeWindow", "orderToBack");
+      return false;
+    }
+
+    // Brings this window in front of any other visible windows.
+    public function orderToFront():Boolean
+    {
+      stub_method("flash.display.NativeWindow", "orderToFront");
+      return false;
+    }
+
+    // Restores this window from either a minimized or a maximized state.
+    public function restore():void
+    {
+      stub_method("flash.display.NativeWindow", "restore");
+    }
+
+    // Starts a system-controlled move of this window.
+    public function startMove():Boolean
+    {
+      stub_method("flash.display.NativeWindow", "startMove");
+      return false;
+    }
+
+    // Starts a system-controlled resize operation of this window.
+    public function startResize(edgeOrCorner:String = "BR"):Boolean
+    {
+      stub_method("flash.display.NativeWindow", "startResize");
+      return false;
+    }
+
+    public function get active():Boolean
+    {
+      stub_getter("flash.display.NativeWindow", "active");
+      return true;
+    }
+
+    public function get closed():Boolean
+    {
+      return this._closed;
+    }
+
+    public function get displayState():String
+    {
+      stub_getter("flash.display.NativeWindow", "displayState");
+      return "normal";
+    }
+
+    [API("668")]
+    public function get isSupported():Boolean
+    {
+      stub_getter("flash.display.NativeWindow", "isSupported");
+      return false;
+    }
+
+    [API("671")]
+    public function get owner():NativeWindow
+    {
+      stub_getter("flash.display.NativeWindow", "owner");
+      return this;
+    }
+
+    [API("675")]
+    public function get renderMode():String
+    {
+      stub_getter("flash.display.NativeWindow", "renderMode");
+      return "auto";
+    }
+    
+    public function get supportsMenu():Boolean
+    {
+      stub_getter("flash.display.NativeWindow", "supportsMenu");
+      return false;
+    }
+
+    public function get supportsNotification():Boolean
+    {
+      stub_getter("flash.display.NativeWindow", "supportsNotification");
+      return false;
+    }
+
+    [API("661")]
```

Comment:
nit: Since the entire class is versioned at this level, there's no point in adding the same limit to any member.